modular_skyrat/modules/mold/code/_mold_defines.dm

```diff
@@ -2,3 +2,37 @@
 
 #define MOLD_TIER_LOW_THREAT 1
 #define MOLD_TIER_HIGH_THREAT 2
+
+#define MAX_MOLD_FOAM_RANGE_BULB 7
+#define MAX_MOLD_FOAM_RANGE_CORE 4
+
+#define TEMP_REAGENT_HOLDER_CAPACITY_LARGE 300
+#define TEMP_REAGENT_HOLDER_CAPACITY_SMALL 210
+
+// Disease
+#define DISEASE_PUFF_RANGE_BULB 4
+#define DISEASE_PUFF_RANGE_CORE 5
+
+#define PUFF_REAGENT_AMOUNT 50
+#define PUFF_REAGENT_EFFICIENCY 24
+
+// EMP
+#define ELECTRICAL_DISCHARGE_HEAVY_RANGE 5
+#define ELECTRICAL_DISCHARGE_LIGHT_RANGE 7
+#define ELECTRICAL_DISCHARGE_SPARKS_AMOUNT 3
+
+#define EMP_SEVERE_EFFECT_CHANCE 50
+#define EMP_ZAP_RANGE 4
+#define EMP_ZAP_POWER 10000
+
+#define EMP_STUN_LENGTH 2 SECONDS
+#define EMP_STUN_RANGE 3
+#define EMP_SOUNDBANG_INTENSITY 1
+#define EMP_SOUNDBANG_STUN_POWER 20
+#define EMP_SOUNDBANG_DAMAGE_POWER 10
+#define EMP_SOUNDBANG_DEAFEN_POWER 5
+
+// Radiation
+#define RAD_PULSE_RANGE 1500
+#define RAD_IRRADIATE_THRESHOLD_BULB 15
+#define RAD_IRRADIATE_THRESHOLD_CORE 10
```

Comment:
The stuff that's only pertaining to `mold.dm` could stay in `mold.dm`, so it can be undefined at the end of the file, and thus doesn't pollute the global space.